.cloudbuild/scripts/cmd/integration-tests/main.go

```diff
@@ -51,15 +54,21 @@ type commandlineArgs struct {
 	targetBranch string
 	commitSHA    string
 	skipChown    bool
+	buildID      string
+	artifacts    customflag.StringArray
+	bucket       string
 }
 
 func parseCommandLine() (commandlineArgs, error) {
 	args := commandlineArgs{}
 
-	flag.StringVar(&args.workspace, "w", "", "Fully-qualified path to the build workspace")
-	flag.StringVar(&args.targetBranch, "t", "", "The PR's target branch")
-	flag.StringVar(&args.commitSHA, "c", "", "The PR's latest commit SHA")
+	flag.StringVar(&args.workspace, "workspace", "", "Fully-qualified path to the build workspace")
```

Comment:
Maybe:

```suggestion
	flag.StringVar(&args.workspace, "workspace", "/workspace", "Fully-qualified path to the build workspace")
```

and simplify scripts?

Reply:
Done